src/account.ts

```diff
@@ -140,6 +140,13 @@ export class Account {
 
         const flatLogs = [result.transaction_outcome, ...result.receipts_outcome].reduce((acc, it) => acc.concat(it.outcome.logs), []);
         this.printLogs(signedTx.transaction.receiverId, flatLogs);
+        // Find all failures in receipts, print out all except last which is handled later.
+        const receiptFailures = result.receipts_outcome.filter(o => typeof o.outcome.status === 'object' && typeof o.outcome.status.Failure === 'object').map(o => o.outcome.status.Failure);
+        if (receiptFailures.length > 1) {
+            for (let i = 0; i < receiptFailures.length - 1; i++) {
```

Comment:
if `receiptFailures` is always present, then you can skip the `if`, since you'd be iterating an empty set which is perfectly fine

and you can probably use `receiptFailures.forEach(receiptFailure => {` instead of the `for`

Reply:
The original idea here was `> 1` instead of `> 0` as the final error appears in the console. This was like, "spit out all the errors except that last one, which you'll see later" like the comment above